docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +231,13 @@ test
 
 If the start page is not specified, it defaults to `about:blank`.
 
+To test webpages in local directories, you can use the `file://` protocol.
+
+```js
+fixture `MyFixture`
```

Comment:
You can use relative paths for pages only in legacy API if you have specified `baseUrl` in `test_config.json`. Currently you can only skip protocol from URL, in this case it's assumed to be  `http://`. So relative file-protocol URLs certainly don't work now.